ml-lifecycle/feature_store/FS_demo.ipynb

```diff
@@ -24,7 +24,6 @@
    "source": [
```

Comment:
<p>No need to add these <code>n</code>s  - keep "a unique ID", "identifier", and "a unique ID" below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>